app/sports/models.py

```diff
@@ -2,7 +2,7 @@
 
 
 class Sport(models.Model):
-    name = models.CharField(max_length=40, blank=False, null=False)
+    name = models.CharField(max_length=40, blank=False, null=False, unique=True)
```

Comment:
Kan ikke huske å ha gjort det her? Ikke med vilje i alle fall

Reply:
Jeg la til dette, men pga. endringene Odd har gjort er det ikke lenger behov for "unique=True"